### PR TITLE
fix(setup-wallet): open wallet when is created/restored

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Open wallet when is created or restored

##### Ticket

YOMO-1440

> [!NOTE]
> Please create the ticket if missing it.

### CODE REVIEW

File: apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts [hunk 1]
```diff
@@ -1108,3 +1109,44 @@ export const useThemeStorageMaker = () => {
...
+  const mutation = useMutation({
+    ...options,
+    mutationFn: (walletMeta) => Promise.all([walletManager.openWallet(walletMeta), walletMeta]),
+  })
+
+  return {
+    openWallet: mutation.mutate,
+    ...mutation,
+  }
+}
```
Comment (banklesss) [inline apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts:1152]: why was it removed?

**Review by jorbuedo — COMMENTED** [review]
Restoring a wallet from the dev options causes the crash

**Comment by banklesss** [comment]
> Restoring a wallet from the dev options causes the crash

Ah!  I thought you mean another problem we had. I will take a look thanks

**Comment by banklesss** [comment]
fixed here https://github.com/Emurgo/yoroi/pull/3276

**Pull request closed without merging**